src/app/ota_image_tool.py

```diff
@@ -219,15 +219,33 @@ def parse_header(args: object):
     """
 
     with open(args.image_file, 'rb') as file:
-        fixed_header_format = '<IQI'
-        fixed_header = file.read(struct.calcsize(fixed_header_format))
+        fixed_header = file.read(struct.calcsize(FIXED_HEADER_FORMAT))
         magic, total_size, header_size = struct.unpack(
-            fixed_header_format, fixed_header)
+            FIXED_HEADER_FORMAT, fixed_header)
         header_tlv = TLVReader(file.read(header_size)).get()['Any']
 
         return magic, total_size, header_size, header_tlv
 
 
+def full_header_size(args: object) -> int:
+    """
+    Returns the size of the fixed header + header
+    """
+    _magic, _total_size, header_size, _header_tlv = parse_header(args)
+    return struct.calcsize(FIXED_HEADER_FORMAT) + header_size
+
+
+def remove_header(args: object) -> None:
+    """
+    Removes the header from args.image_file and writes to args.output_file
+    """
+    image_start = full_header_size(args)
+    with open(args.image_file, 'rb') as file:
+        with open(args.output_file, 'wb') as outfile:
+            file.seek(image_start)
+            outfile.write(file.read())
```

Comment:
How large can images get? Since this is embedded, I wonder if a few MB is always the max size or can they get actually large (e.g. non-stripped debug builds)?

Reply:
Most embedded platforms will use images less than a MB, but some may provide complete Linux distros (there are some yocto integrations in the repo), so I guess they can be bigger (not sure how big exactly).